src/pages/profile/_id/index.vue

```diff
@@ -281,6 +281,12 @@ export default {
       }
       return '0'
     },
+    getTotalPlayTimeText(time) {
+      let hours = Math.floor(time / 36000)
+      let minutes = Math.floor((time % 36000) / 600)
+      let seconds = Math.floor((time % 600) / 10)
+      return `${hours}:${minutes}:${seconds}`
```

Comment:
This could lead to single digits (as in: `3:5:8`, which would usually be formatted like `3:05:08`). Maybe pad minutes and seconds?
```suggestion
      return `${hours}:${String(minutes).padStart(2, "0")}:${String(seconds).padStart(2, "0")}`
```
(Maybe write that neater because my code is terrible lol)